src/kernel/bitcoinkernel.cpp

```
@@ -0,0 +1,45 @@
+        remainder = -remainder;
+    }
+    return UniValue(UniValue::VNUM,
+            strprintf("%s%d.%08d", amount < 0 ? "-" : "", quotient, remainder));
+}
+//////////////////////
+
+std::string GetPrettyExceptionStr(const std::exception_ptr& e)
+{
+    try {
+        // rethrow and catch the exception as there is no other way to reliably cast to the real type (not possible with RTTI)
+        std::rethrow_exception(e);
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)